winbuild/build_prepare.py

```diff
@@ -469,11 +469,22 @@ def extract_dep(url, filename):
             raise RuntimeError(ex)
 
     print("Extracting " + filename)
+    sources_dir_abs = os.path.abspath(sources_dir)
     if filename.endswith(".zip"):
         with zipfile.ZipFile(file) as zf:
+            for member in zf.namelist():
+                member_abspath = os.path.abspath(os.path.join(sources_dir, member))
+                member_prefix = os.path.commonprefix([sources_dir_abs, member_abspath])
```

Comment:
```suggestion
                member_prefix = os.path.commonpath([sources_dir_abs, member_abspath])
```
Unless I'm missing something, this would mean that you couldn't go outside of the `src` directory to `src2`.

Reply:
Good point. I'm guessing the @TrellixVulnTeam bot can't tell whether the project requires Python 3.5 where this function was added.